.github/workflows/release-fastlane-android.yml

```diff
@@ -44,15 +48,10 @@ jobs:
       - uses: webfactory/ssh-agent@v0.9.0
         with:
           ssh-private-key: ${{ steps.google-secrets.outputs.BOT_SSH_KEY }}
```

Comment:
Cool, I think we can remove this one too. It was needed when we had the branding.